Services/DeviceClient.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services
     public interface IDeviceClient
     {
         IoTHubProtocol Protocol { get; }
+        string DeviceId { get; }
```

Comment:
I don't t think we need this

Reply:
Updated connection actor as per your suggestion, however, for telemetry actor, how do I get deviceId ?

Reply:
yeah unfortunately it's not possible right now - maybe we'll address this later 👍